src/fmath.h

```diff
@@ -38,12 +40,17 @@ class PID_Controller{
 		fs_PidFloat_t kp, ki, kd;
 		//error_prevtwo, calls
 		fs_PidFloat_t output_past, error, error_prev, error_sum;
+    
+    private:
+        // Adicione um membro para armazenar o objeto PIDParameters
+        PIDParameters parameters;
+};
 	
 };
 #ifdef __LART_KINEMATICS_DOUBLE_PRECISION__
```

Comment:
These "if" clauses are redundant, and thus can be removed. In any case long double is used.